keyboards/sawnsprojects/re65/info.json

```
@@ -0,0 +1,142 @@
+{
+    "keyboard_name": "RE65",
+    "maintainer": "Salane",
+    "manufacturer": "Mai The San",
+    "url": "",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "usb": {
+        "vid": "0x534C",
+        "pid": "0x0C65",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": true,
+        "rgblight": true,
+        "encoder": true,
+        "encoder_map": true
+    },
+    "dynamic_keymap": {
+        "layer_count": 6
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["GP27", "GP26", "GP25", "GP29", "GP4"],
+        "cols": ["GP20", "GP19", "GP18", "GP24", "GP23", "GP22", "GP17", "GP28", "GP2", "GP3", "GP12", "GP11", "GP10", "GP9", "GP8"]
+    },
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP5", "pin_b": "GP6"}
+        ]
+    },
+    "rgblight": {
+        "led_count": 20,
+        "hue_steps": 8,
+        "saturation_steps": 8,
+        "brightness_steps": 8,
+        "max_brightness": 255,
+        "sleep": true,
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        }
+    },
+    "ws2812": {
+        "pin": "GP21",
+        "driver": "vendor"
+    },
+    "layouts": {
+        "LAYOUT": {
```

community layout support added
```suggestion
    },
    "community_layouts": ["65_ansi_blocker"],
    "layouts": {
        "LAYOUT_65_ansi_blocker": {
```